tests/sentry/tasks/test_weekly_escalating_forecast.py

```diff
@@ -41,7 +45,9 @@ def test_empty_escalating_forecast(
             run_escalating_forecast()
             group = group_list[0]
             fetched_forecast = EscalatingGroupForecast.fetch(group.project.id, group.id)
-            assert fetched_forecast is None
+            assert fetched_forecast is not None
+            if fetched_forecast:
```

Comment:
not sure the if here is necessary

Reply:
is it a type thing?

Reply:
yeah, typing fails without it

Reply:
you could try something like this but what you have is fine! 
```suggestion
            assert fetched_forecast and fetched_forecast.forecast == ONE_EVENT_FORECAST
```